ddtrace/llmobs/_llmobs.py

```diff
@@ -1402,6 +1431,34 @@ def activate_distributed_headers(cls, request_headers: Dict[str, str]) -> None:
         error = cls._instance._activate_llmobs_distributed_context(request_headers, context)
         telemetry.record_activate_distributed_headers(error)
 
+    @classmethod
+    def prompt_context(
+        cls,
+        name: Optional[str] = None,
+        version: str = "1.0.0",
+        prompt_id: Optional[str] = None,
+        template: Optional[str] = None,
+        chat_template: Optional[Union[List[Dict[str, str]], List[Message]]] = None,
+        variables: Optional[Dict[str, Any]] = None,
+        rag_context_variable_keys: Optional[List[str]] = None,
+        rag_query_variable_keys: Optional[List[str]] = None,
+    ) -> AnnotationContext:
+        """
+        shortcut to create a prompt or pass a prompt object and annotate it
```

Comment:
```suggestion
        Creates an annotation context with a `Prompt` constructed on the following
```
and then maybe add docstrings for each of the prompt characteristics that are taken as arguments